scripts/fbi/hate_crime/geo_id_resolver.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Resolving place dcid given the Pub_agency_name, Agency_type_name, State_Abbr
+from FBI hate crime dataset
+"""
+
+import os
+import sys
+import csv
+
+# Allows the following module imports to work when running as a script
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH,
+                             '../../../util'))  # For State and County map
+
+from alpha2_to_dcid import USSTATE_MAP
+from county_to_dcid import COUNTY_MAP
+
+city_geocodes_csv_path = os.path.join(_SCRIPT_PATH,
+                                      '../crime/city_geocodes.csv')
+manual_city_geocodes_csv_path = os.path.join(_SCRIPT_PATH,
+                                             '../crime/manual_geocodes.csv')
+
+_US_GEO_CODE_UPDATE_MAP = {
+    # Mapping geos in data to geos in csv / map
+    'NB': 'NE',
+    'GM': 'GU'
+}
+
+_IGNORE_STATE_ABBR = ['FS']  # Ignoring federal codes
+_IGNORE_CITIES = ['abington township pa',
+                  'lone tree co']  # ignoring cities which have duplicates
+
+city = {}
```

Comment:
module level constant should be all caps and prefixed with an underscore
https://google.github.io/styleguide/pyguide.html#254-decision

Reply:
Done.